src/metabase/api/common.clj

```diff
@@ -261,16 +262,50 @@
                                   (ns-name *ns*) fn-name)))
       (assoc parsed :fn-name fn-name, :route route, :docstr docstr))))
 
+(defn validate-param-values
```

Comment:
Looking at this name, I expect it to throw an exception if the param values are not valid.